google-cloud-tools-plugin/google-cloud-core/testSrc/com/google/cloud/tools/intellij/project/ProjectSelectionDialogTest.java

```diff
@@ -79,7 +79,6 @@ public void setUp() {
     doNothing().when(projectSelectionDialog).installTableSpeedSearch(any());
     doNothing().when(projectSelectionDialog).setLoading(anyBoolean());
 
-    projectSelectionDialog.createUIComponents();
```

Comment:
Oh this one will get called by IDEA form instrumentation?

Reply:
right. the default constructor (automatically invoked due to @Spy annotation) includes this in its form generated code.